esmvaltool/recipes/cmorizers/recipe_era5-land.yml

```diff
@@ -0,0 +1,32 @@
+# ESMValTool
+# recipe_era5.yml
+---
+documentation:
+  description: CMORize ERA5-Land data
+
+  authors:
+    - andela_bouwe
+    - kalverla_peter
+
+  references:
+    - acknow_project
```

Comment:
```suggestion
    - era5
```

Reply:
@katjaweigel I suggested adding `era5` tag to the reference section. However, I noticed that era5-land has its own reference that can be found [here](https://cds.climate.copernicus.eu/cdsapp#!/dataset/10.24381/cds.68d2bb30?tab=overview), with a DOI: 10.24381/cds.68d2bb30. Could you please create a `era5land.bibtex` file and add it to `esmvaltool/references` folder. You can use  https://doi2bib.org/ to convert the doi to BibTeX and find more information on how to do it in the documentation, [here](https://docs.esmvaltool.org/en/latest/community/diagnostic.html?highlight=reference#adding-references). Please let me know if it is not clear.

Reply:
Thanks, I added it. (and I really like the  https://doi2bib.org/ page)